nautobot/dcim/tests/test_filters.py

```diff
@@ -701,6 +722,282 @@ def common_test_data(cls):
+            positions=2,
+            label="rearport2",
+            description="Rear Port Description 2",
+        ),
+        RearPortTemplate.objects.create(
+            module_type=module_types[2],
+            name="Module Rear Port 3",
+            type=PortTypeChoices.TYPE_BNC,
+            positions=3,
+            label="rearport3",
+            description="Rear Port Description 3",
+        ),
+    )
+
+    FrontPortTemplate.objects.create(
+        module_type=module_types[0],
+        name="Module Front Port 1",
+        rear_port_template=rear_ports[0],
+        type=PortTypeChoices.TYPE_8P8C,
+        rear_port_position=1,
+        label="frontport1",
+        description="Front Port Description 1",
+    )
+    FrontPortTemplate.objects.create(
+        module_type=module_types[1],
+        name="Module Front Port 2",
+        rear_port_template=rear_ports[1],
+        type=PortTypeChoices.TYPE_110_PUNCH,
+        rear_port_position=2,
+        label="frontport2",
+        description="Front Port Description 2",
+    )
+    FrontPortTemplate.objects.create(
+        module_type=module_types[2],
+        name="Module Front Port 3",
+        rear_port_template=rear_ports[2],
+        type=PortTypeChoices.TYPE_BNC,
+        rear_port_position=3,
+        label="frontport3",
+        description="Front Port Description 3",
+    )
+
+    ModuleBayTemplate.objects.create(
+        module_type=module_types[0],
+        position="test module bay 1",
+        label="modulebay1",
+        description="test module bay 1 description",
+    )
+    ModuleBayTemplate.objects.create(
+        module_type=module_types[1],
+        position="test module bay 2",
+        label="modulebay2",
+        description="test module bay 2 description",
+    )
+    ModuleBayTemplate.objects.create(
+        module_type=module_types[2],
+        position="test module bay 3",
+        label="modulebay3",
+        description="test module bay 3 description",
+    )
+    module_bay = ModuleBay.objects.filter(installed_module__isnull=True).first()
+    module_status = Status.objects.get_for_model(Module).first()
+    parent_module = Module.objects.create(
+        parent_module_bay=module_bay,
+        module_type=module_types[0],
+        status=module_status,
+        tenant=tenants[0],
+    )
+    Module.objects.create(
+        parent_module_bay=parent_module.module_bays.first(),
+        module_type=module_types[1],
+        status=module_status,
+        tenant=tenants[1],
+    )
+    other_parent_module = Module.objects.create(
+        location=Location.objects.get_for_model(Module).first(),
+        module_type=module_types[2],
+        status=module_status,
+        tenant=tenants[2],
+    )
+    Module.objects.create(
+        parent_module_bay=other_parent_module.module_bays.first(),
+        module_type=module_types[2],
+        status=module_status,
+        tenant=tenants[2],
+    )
+
+
+class ComponentTemplateMixin:
```

Comment:
Should all of these mixins be named `....TestMixin` to make them less ambiguous?